### ISSUE TITLE
Settings editor entries jump up when interacted with

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#134001]
* open setting UI
* type "inlay"
* scroll so that a setting is half revealed
* select/unselect it
* :bug: the setting jumps up, it does _not_ always select/unselect the item

(_sometimes needs a few tries/scrolls to reproduce..._)

https://user-images.githubusercontent.com/1794099/135087770-8a4f6a67-5323-439c-967b-353596989d61.mov

**Comment by roblourens** [comment]
Something to do with these inlay settings and their dumb async rendered descriptions.

**Comment by roblourens** [comment]
The dynamic heights of the rows with this markdown are not handled correctly, we have to rethink this. For example, these rows will start overlapping if you resize the editor.

**Comment by mitsukuri** [comment]
For me, this happens not _sometimes_, but **all the time**, to the point that it's effectively impossible to edit settings via the editor, forcing me to edit JSON directly.

**Comment by WebMechanic** [comment]
I second that. 
Some recent update causes the right pane in the UI to jump, "jump scroll" upwards in roughly 40px increments and on occasion see the cursor/focus hopping fields as well as between the two panels:  the outline changes on the active field/pane.

It randomly works but for the most times the UI became unusable.

**Comment by abelgladstone** [comment]
I can add to this conversation that this issue pops up if I have the following setting in my settings.json file.

```
"terminal.integrated.profiles.windows": {
    "PowerShell": {
      "source": "PowerShell",
      "icon": "terminal-powershell"
    },
    "Command Prompt": {
      "path": [
        "${env:windir}\\Sysnative\\cmd.exe",
        "${env:windir}\\System32\\cmd.exe"
      ],
      "args": [],
      "icon": "terminal-cmd"
    },
    "Git": {
      "path": "C:/Git/bin/bash.exe",
      "args": ["-i", "-l"]
    }
  },
"terminal.integrated.defaultProfile.windows": "Git"
```

In my case I have GIT installed in a non standard location. This issue goes away if default profile is not set.

The following is my vscode version.

Version: 1.62.3 (user setup)
Commit: ccbaa2d27e38e5afa3e5c21c1c7bef4657064247
Date: 2021-11-17T08:11:14.551Z
Electron: 13.5.2
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19042

**Comment by mitsukuri** [comment]
I'd like to confirm @abelgladstone's findings, but in my case it doesn't have to do with Git.

Here's the relevant part of my `settings.json`:
```
"terminal.integrated.profiles.windows": {
        "PowerShell": {
            "args": [
                "-nol"
            ],
            "path": "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe"
        },
        "PowerShell Dev": {
            "args": [
                "-nol",
                "-noe",
                "-c",
                "&{Import-Module 'C:/Program Files (x86)/Microsoft Visual Studio/2019/BuildTools/Common7/Tools/Microsoft.VisualStudio.DevShell.dll'; Enter-VsDevShell 2a7a9ed6 -DevCmdArguments '-arch=x64'}"
            ],
            "path": "C:/Windows/System32/WindowsPowerShell/v1.0/powershell.exe"
        }
    },
"terminal.integrated.defaultProfile.windows": "PowerShell Dev",
```
For me, it's enough to comment the `terminal.integrated.defaultProfile.windows` line while leaving `terminal.integrated.profiles.windows` entry alone, and it fixes the buggy behavior of settings editor immediately.

**Comment by rzhao271** [comment]
For the Settings editor issue related to the terminal profile settings, see https://github.com/microsoft/vscode/issues/136509.
@jrieken does your original issue also go away when commenting out the terminal profile setting(s)?

**Comment by abelgladstone** [comment]
This problem seems to be fixed for me in version 1.63.2

**Comment by mitsukuri** [comment]
1.63.2 fixes this for me too, yay!
